code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -180,21 +174,10 @@
 					return
 				new /mob/living/simple_animal/corgi/puppy(loc)
 
-
 		if(prob(1))
 			visible_emote(pick("dances around","chases her tail"))
 			for(var/i in list(1,2,4,8,4,2,1,2,4,8,4,2,1,2,4,8,4,2))
 				set_dir(i)
 				sleep(1)
 				if(QDELETED(src) || client)
 					return
-
-/mob/living/simple_animal/corgi/harvest_skin()
-	. = ..()
-	. += new/obj/item/corgi_hide(get_turf(src))
-
-/obj/item/corgi_hide
```

Comment:
Noo, not the trophies

Reply:
Can readd them if you like, they didn't seem to be used anywhere.

Reply:
It's just trophy to show you skinned runtime / ian, because there's special icons for cat/corgi hide but material stacks can't support that

Reply:
Readded em